Explorer/Assets/DCL/Settings/SettingsDataStore.cs

```diff
@@ -2,47 +2,45 @@
 
 namespace DCL.Settings
 {
-    public class SettingsDataStore : ISettingsDataStore
+    public class SettingsDataStore
```

Comment:
I think we should remove this class altogether, all it really does is forward calls to DCLPlayerPrefs 🤔 I'd maybe also add Get/SetBool to it so we don't have to use ints (since we can do that now directly).

Reply:
Done!